numismatic/events.py

```diff
@@ -7,10 +7,12 @@
 
 
 class OrderType(str, Enum):
+    TRADE = 'TRADE'
```

Comment:
:+1:  I was going to suggest this

Reply:
Orders should be BID and ASK, only actual trades are BUY or SELL. I've used this incorrectly somewhere and this needs to be fixed.